lib/node_modules/@stdlib/math/base/special/cidentity/manifest.json

```diff
@@ -24,15 +26,51 @@
 	],
 	"confs": [
 		{
-			"src": [
-				"./src/main.c"
-			],
-			"include": [
-				"./include"
-			],
-			"libraries": [],
-			"libpath": [],
-			"dependencies": []
+		  "task": "build",
```

Comment:
@steff456 The indentation in this file is mixed tabs and spaces. Should be normalized according to EditorConfig conventions.